docs/manual/key-storage/storage-plugins/aws-secrets-manager.md

```diff
@@ -13,6 +13,31 @@ This integration allows users to retrieve secrets from the AWS Secrets Manager s
 The AWS Secrets integration uses authentication set through the AWS Plugin within System Configuration. If this is already configured, continue on to the next section.<br>
 Otherwise, follow the instructions [**here**](/manual/plugins/aws-plugins-overview.html#setup) and choose the auth type that aligns with your Runbook Automation deployment type.
 
+### IAM Permissions 
```

Comment:
These only look like Read Access.  Does this allow writing secrets via Rundeck as well?

Reply:
yes - pushed commit to explain those permissions as well